### ISSUE TITLE
BIGIP restart restnoded

### ISSUE THREAD

**Issue by luckymuk** [F5Networks/f5-appsvcs-extension#65]
##### Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

##### ISSUE TYPE

 - Bug Report

##### AS3 BUILD/ VERSION

```
3.8.1
```

##### BIGIP VERSION

```
14.1.0 Build 0.0.116
```

##### SUMMARY

```
I am using cURL to send API request for configuring BIGIP virtual server. I noticed that when there is a mistake in the body of cURL BIG IP restart restnoded. Here is the error message.
 
Broadcast message from systemd-journald@ltm-lak.local.com (Thu 2019-02-14 06:26:45 PST):
 
logger[11819]: Re-starting restnoded
 ```

##### STEPS TO REPRODUCE

```
{
   "class": "AS3",
   "action": "deploy",
   "persist": true,
   "declaration": {
      "class": "ADC",
      "schemaVersion": "3.0.0",
      "id": "urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d",
      "label": "Sample 1",
      "remark": "Simple HTTP application with RR pool",
      "Sample_01": {
         "class": "Tenant",
         "A1": {
            "class": "Application",
            "template": "http",
            "serviceMain": {
               "class": "Service_HTTP",
               "virtualAddresses": [
                  "10.0.1.10"
               ],
               "pool": "web_pool"
            },
            "web_pool": {
               "class": "Pool",
               "monitors": [
                  "http"
               ],
               "members": [{
                  "servicePort": 80,
                  "serverAddresses": [
                     "192.0.1.10",
                     "192.0.1.11"
                  ]
               }]
            }
         }
      }
   }
}
```
Here is my cURL call
curl -k -u $PASS -X POST https://{BIGIP-IPADDRESS}/mgmt/shared/appsvcs/declare -d sample.json
 
Right cURL syntax (I get expected behavior with this API call)
curl -k -u $PASS -X POST https://{BIGIP-IPADDRESS}/mgmt/shared/appsvcs/declare -d @sample.json
```

<!--- You can also paste gist.github.com links for larger files -->

##### EXPECTED RESULTS
<!--- What did you expect to happen when running the steps above? -->

```
BIGIP shouldn't start the restnoded
```

##### ACTUAL RESULTS
<!--- What actually happened?  -->

<!--- Paste verbatim command output between quotes below -->
```
Broadcast message from systemd-journald@ltm-lak.local.com (Thu 2019-02-14 06:26:45 PST):
 
logger[11819]: Re-starting restnoded
```

**Comment by dstokesf5** [comment]
Thank you for the bug report, we are now tracking the issue internally.

**Comment by kaustriaf5** [comment]
Tracked internally as 900, slated fix in 3.10

**Comment by kaustriaf5** [comment]
@luckymuk please see release [3.10](https://github.com/F5Networks/f5-appsvcs-extension/releases/tag/v3.10.0) for the fix. Thanks!
